### PR TITLE
Adding Footer Component

### LINKED ISSUE #11: Make Footer Component

### PR DESCRIPTION
**Pull request by kashika3101**
# Description

Added the Footer component

Fixed #11 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/commons/CSS/Footer.module.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    #type3,
+    #type1,
+    #type5 {
+      font-size: 13px;
+    }
+  }
+  @media (min-width: 991px) and (max-width: 1440px) {
+    .icon {
+      float: right;
+
```
Comment (vinitshahdeo) [inline src/commons/CSS/Footer.module.css]: @kashika3101 Remove extra lines.

File: src/commons/Components/Footer.js [hunk 2]
```diff
@@ -1,5 +1,70 @@
 import React from "react";
+import styles from "../CSS/Footer.module.css";
+import { GrInstagram } from "react-icons/gr";
+import { FaTwitterSquare } from "react-icons/fa";
+import { GrLinkedin } from "react-icons/gr";
+
```
Comment (vinitshahdeo) [inline src/commons/Components/Footer.js]: @kashika3101 Remove extra new lines.

File: src/commons/Components/Footer.js [hunk 3]
```diff
@@ -1,5 +1,70 @@
...
+        </div>
+
+        <hr />
+        <div className="row">
+          <div className="col" id={styles.s}>
+            &copy; {new Date().getFullYear()} Girl Code It | All Rights Reserved
+          </div>
+         
+          <div className="col-md-4 col-12 ">
+            <GrInstagram className={styles.icon} />
```
Comment (vinitshahdeo) [inline src/commons/Components/Footer.js]: Why aren't we using Font Awesome here too?

**Review by vinitshahdeo — COMMENTED** [review]
@kashika3101 Added a few comments.

File: src/commons/Components/Footer.js (hunk 3 above)
Comment (kashika3101) [reply src/commons/Components/Footer.js]: As per the recommendations, we are now using react icons in the entire footer components.

**Review by Manvityagi — CHANGES_REQUESTED** [review]
It is affecting other components on the screen.

**Pull request closed without merging**